proxy/frontend/core/src/main/java/org/apache/shardingsphere/proxy/frontend/netty/CDCChannelInboundHandler.java

```diff
@@ -164,69 +163,79 @@ private String getHostAddress(final ChannelHandlerContext context) {
                     .addListener(ChannelFutureListener.CLOSE);
             return;
         }
-        checkPrivileges(connectionContext.getCurrentUser().getGrantee(), startSubscriptionRequest.getDatabase());
-        CDCResponse response = backendHandler.startSubscription(request, ctx.channel(), connectionContext);
+        String database = backendHandler.getDatabaseByJobId(startStreamingRequestBody.getStreamingId());
+        checkPrivileges(connectionContext.getCurrentUser().getGrantee(), database);
+        CDCResponse response = backendHandler.startStreaming(request.getRequestId(), startStreamingRequestBody.getStreamingId(), connectionContext, ctx.channel());
         ctx.writeAndFlush(response);
     }
     
-    private void processStopSubscription(final ChannelHandlerContext ctx, final CDCRequest request, final CDCConnectionContext connectionContext) {
-        StopSubscriptionRequest stopSubscriptionRequest = request.getStopSubscription();
-        checkPrivileges(connectionContext.getCurrentUser().getGrantee(), stopSubscriptionRequest.getDatabase());
-        backendHandler.stopSubscription(connectionContext.getJobId());
+    private void processStopStreamingRequest(final ChannelHandlerContext ctx, final CDCRequest request, final CDCConnectionContext connectionContext) {
+        StopStreamingRequestBody stopStreamingRequestBody = request.getStopStreamingRequestBody();
+        String database = backendHandler.getDatabaseByJobId(stopStreamingRequestBody.getStreamingId());
+        checkPrivileges(connectionContext.getCurrentUser().getGrantee(), database);
+        backendHandler.stopStreaming(connectionContext.getJobId());
         connectionContext.setStatus(CDCConnectionStatus.LOGGED_IN);
+        connectionContext.setJobId(null);
         ctx.writeAndFlush(CDCResponseGenerator.succeedBuilder(request.getRequestId()).build());
     }
     
-    private void processDropSubscription(final ChannelHandlerContext ctx, final CDCRequest request, final CDCConnectionContext connectionContext) {
-        DropSubscriptionRequest dropSubscriptionRequest = request.getDropSubscription();
-        checkPrivileges(connectionContext.getCurrentUser().getGrantee(), dropSubscriptionRequest.getDatabase());
+    private void processDropStreamingRequest(final ChannelHandlerContext ctx, final CDCRequest request, final CDCConnectionContext connectionContext) {
+        DropStreamingRequestBody dropStreamingRequestBody = request.getDropStreamingRequestBody();
+        String database = backendHandler.getDatabaseByJobId(dropStreamingRequestBody.getStreamingId());
+        checkPrivileges(connectionContext.getCurrentUser().getGrantee(), database);
         try {
-            backendHandler.dropSubscription(connectionContext.getJobId());
+            backendHandler.dropStreaming(connectionContext.getJobId());
+            connectionContext.setStatus(CDCConnectionStatus.LOGGED_IN);
```

Comment:
Is it necessary to set status as `LOGGED_IN` when dropStreaming? And also when stopStreaming

Reply:
Remove `STREAMING`, this doesn't seem to be needed at the moment